src/components/NewLocationPage/NotesBlock/NotesBlock.style.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import { COLOR_MAP } from 'common/colors';
+
+export const SectionContentContainer = styled.div`
+  display: flex;
+  font-size: initial;
+  text-transform: none;
```

Comment:
[once we implement StyledBaseButton](https://github.com/covid-projections/covid-projections/pull/3403/files#r618804311), we can remove `font-size: initial;` and `text-transform: none;`